### PR TITLE
PLT-4449 Update placeholder texts for channel header and purpose

### PR DESCRIPTION
**Pull request by jasonblais**
#### Summary
Update placeholder texts for channel header and purpose in channel creation dialog

#### Ticket Link
Second part of [PLT-4449](https://mattermost.atlassian.net/browse/PLT-4449)

#### Checklist
- [X] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates

### CODE REVIEW

**Comment by jasonblais** [comment]
Tested on Windows 10, Chrome (including mobile view)

image

File: webapp/components/new_channel_modal.jsx [hunk 1]
```diff
@@ -20,6 +20,14 @@ const holders = defineMessages({
...
         id: 'channel_modal.nameEx',
         defaultMessage: 'E.g.: "Bugs", "Marketing", "客户支持"'
+    },
+    purposeEx: {
+        id: 'channel_modal.purposeEx',
+        defaultMessage: 'E.g.: "A channel to file bugs and improvements"'
+    },
+    headerEx: {
+        id: 'channel_modal.headerEx',
+        defaultMessage: 'E.g.: "[Link Title](http://example.com)"'
```
Comment (crspeller) [inline webapp/components/new_channel_modal.jsx]: Can we remove the injectIntl stuff here? I know it's not related to this PR, but we should be removing it when we see it.

Comment (crspeller) [inline webapp/components/new_channel_modal.jsx]: Oh wait, never mind. I thought this was a dev.

**Comment by crspeller** [comment]
@jasonblais Can you assign this ticket to a dev with a comment telling them this branch has the start of the work on it? I want to get this injectIntl stuff removed.

**Pull request closed without merging**